base/commands/project/utils.go

```diff
@@ -4,52 +4,45 @@ import (
 	"errors"
 	"fmt"
 	"os"
-	"strings"
 
 	"github.com/hazelcast/hazelcast-commandline-client/clc/paths"
 	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
 	"github.com/hazelcast/hazelcast-commandline-client/internal/str"
 	"gopkg.in/yaml.v2"
 )
 
-func loadFromDefaults(templateDir string, props *map[string]string) error {
+func loadFromDefaults(templateDir string) (map[string]string, error) {
+	props := make(map[string]string)
 	path := paths.Join(templateDir, defaultsFileName)
 	if !paths.Exists(path) {
-		return nil
+		return nil, nil
 	}
 	b, err := os.ReadFile(path)
 	if err != nil {
 		if errors.Is(err, os.ErrNotExist) {
-			return nil
+			return nil, nil
 		}
```

Comment:
Since the path existence is checked in the code above `os.ReadFile`, can this check be removed?

Reply:
I removed path existence check instead, so it doesn't seem like error is not handled in here.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/287/commits/ac5a906e25afdbb22dad1ebf035683c9b0263155